### ISSUE TITLE
Functions from 'sa-east-1' are not reachable in this region ('us-east-1')

### ISSUE THREAD

**Issue by leandrock** [alexa-labs/ask-cli#46]
## I'm submitting a...

<pre><code>
[ x ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

When I try to askx deploy my project after updating to ask v2, I receive this region error message. How do I configure the askx to deploy to sa-east-1 region? Thanks.

## Your Environment and Context

* ask-cli version: latest (used the askx util upgrade-to-v2)
* Operating System and version: Windows 10
* Node.js version used for development: 12.x
* NPM version used for development: 6.13.0

**Comment by ChihYing7509** [comment]
Hello @leandrock,
Thank you for bringing this to our attention. 

Could you provide your **Alexa default region** and **AWS region** for us to figure out the issue? 

Thank you

**Comment by leandrock** [comment]
Hi @ChihYing7509 .

On .aws folder  credentials file I have region=sa-east-1
Not sure how I can check the Alexa default region. Can you assist me on that?

Thank you.

**Comment by leandrock** [comment]
@ChihYing7509 
If by Alexa default region you mean the lambda function region, there you go:
arn:aws:lambda:sa-east-1

So both my local .aws folder (credentials file) and the lambda funcion are set to sa-east-1 region

**Comment by RonWang** [comment]
Hi @leandrock ,
Thanks for your bug report! It seems we don't correctly take in the default userConfig's awsRegion. To unblock you first, please follow the [instruction](https://github.com/alexa-labs/ask-cli/blob/master/docs/concepts/Deploy-Command.md#deployer) and use the `regionalOverrides` feature for your desired AWS region. You can try setting ask-resources config's `skillInfrastructure.userConfig.regionalOverrides` with:
```
{
  "regionalOverrides": {
    "{your_region}": {
      "awsRegion": "sa-east-1"
    }
  }
}
```
This will deploy {your_region}'s Lambda function with the "awsRegion", i.e. "sa-east-1".

We will address this issue later with a fix.

Thanks,
Ron

**Comment by ChihYing7509** [comment]
Fixed it in chore(release): 0.7.2
